src/usecases/read_and_send_test.go

```diff
@@ -101,11 +96,7 @@ func Test_moveFile_SourceUrlDoesNotContainStartingFolder_FileIsNotMoved(t *testi
 }
 
 func Test_moveFile_BlobHandlerFailsToMoveFile_LogsError(t *testing.T) {
-	defaultLogger := slog.Default()
-	defer slog.SetDefault(defaultLogger)
-
-	buffer := &bytes.Buffer{}
-	slog.SetDefault(slog.New(slog.NewTextHandler(buffer, nil)))
```

Comment:
I notice the new `SetupLogger` doesn't reset to the default logger - was that just not necessary before so it was okay to remove?

Reply:
I think it's probably fine (the tests each seem to get a new `buffer` and logger setup when actually running the app happens elsewhere), but would like a quick gut check from @halprin since he set up the original pattern we're changing here

Reply:
The problem with not resetting it (I think, I haven't fully tested this out) is that any test that runs after this one that doesn't want to test the log output won't display any log output.

I'm partial to seeing log output in the tests when I'm not purposely trying to capture the log output.  What do others think?

Reply:
I think you're right that tests run after this don't show log output

Reply:
Added that back in with Sylvie!